src/read/pdb/parser.rs

```diff
@@ -45,11 +54,30 @@ pub fn open_pdb(
 /// If you want to open a file see [`open_pdb`]. There is also a function to open a mmCIF file directly
 /// see [`crate::open_mmcif`] and [`crate::open_mmcif_raw`]. If you want to open a general file
 /// with no knowledge about the file type see [`crate::open`] and [`crate::open_raw`].
+#[deprecated(
+    since = "0.12.0",
+    note = "Please use `ReadOptions::default().read_raw(input, context)` instead"
+)]
 pub fn open_pdb_raw<T>(
     input: std::io::BufReader<T>,
     context: Context,
     level: StrictnessLevel,
 ) -> Result<(PDB, Vec<PDBError>), Vec<PDBError>>
+where
+    T: std::io::Read,
+{
+    open_pdb_raw_with_options(input, context, ReadOptions::default().set_level(level))
+}
+
+/// Parse the input stream into a PDB struct with [`ReadOptions`].
+///
+/// # Related
+/// See [`open_pdb_raw`] for a version of this function with sane defaults.
+pub fn open_pdb_raw_with_options<T>(
```

Comment:
also `pub(crate)`

Reply:
Alternatively, we mark this function as `#[unstable]` so that people can't simply use them and not realise that they may change